v2/cn/download.md.tt2

```diff
@@ -1,146 +1,182 @@
 <!---
     @title         下载
-    @creator       Yichun Zhang
+    @creator       章亦春
+    @Translator    何伟平
```

Comment:
We already track the latest modifier from the git history. But this translator field is not honored by the toolchain :)